MANIFEST.SKIP

```diff
@@ -0,0 +1,6 @@
+\.mailmap
+\.gitignore
+\.gitattributes
+\.git_patch
+^\.git\b
+^\.github/
```

Comment:
missing `\.travis\.yml`

Reply:
I'm deliberately not aiming to change anything about what is shipped and what is not in this branch, that should be done in a different PR

Reply:
this works for me